src/utils/components/AdditionalResources/AdditionalResources.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import VirtualMachineModel from '@kubevirt-ui/kubevirt-api/console/models/VirtualMachineModel';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+} from '@patternfly/react-core';
+
+type AdditionalResourcesProps = {
+  template: V1Template;
+};
+
+const AdditionalResources: React.FC<AdditionalResourcesProps> = ({ template }) => {
+  const { t } = useKubevirtTranslation();
+
+  if (template.objects.length === 1) return null;
+
+  return (
+    <DescriptionListGroup>
+      <DescriptionListTerm>{t('Additional resources')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        <ul>
+          {template.objects
```

Comment:
please add ?. , there is quite a few in this file :)

Reply:
solved thanks